be/src/exec/pipeline/scan_operator.cpp

```diff
@@ -254,6 +258,41 @@ Status ScanOperator::_pickup_morsel(RuntimeState* state, int chunk_source_index)
     return Status::OK();
 }
 
+void ScanOperator::_merge_chunk_source_profiles() {
+    std::vector<RuntimeProfile*> profiles(_chunk_source_profiles.size());
+    for (auto i = 0; i < _chunk_source_profiles.size(); i++) {
+        profiles[i] = _chunk_source_profiles[i].get();
+    }
+    RuntimeProfile::merge_isomorphic_profiles(profiles);
+
+    RuntimeProfile* merged_profile = profiles[0];
```

Comment:
what if it's empty?

Reply:
the number of profiles must equals to MAX_IO_TASKS_PER_OP, which is greater than 0

Reply:
I can add DCHECK here